packages/secrets/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe Secrets SDK package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Reduced number of keychain unlock prompts on MacOS for simultaneous access to secrets by the same process.
```

Comment:
The fix really addresses more than one process. I suggest the following wording:
... by multiple instances of the same application.